packages/hoppscotch-common/src/components/lenses/renderers/HTMLLensRenderer.vue

```diff
@@ -109,8 +110,17 @@ const props = defineProps<{
     | HoppRESTRequestResponse
   isSavable: boolean
   isEditable: boolean
+  tabId: string
 }>()
 
+const { containerRef, scrollerRef } = useScrollerRef(
+  "HTMLLens",
+  ".cm-scroller",
+  undefined, // skip initial
+  `${props.tabId}::html`
+)
+defineExpose({ scrollerRef })
```

Comment:
can you explain the use of defineExpose here, do we need to propagate this to the parent component ?